test/integration/main_test.go

```diff
@@ -236,6 +227,153 @@ func forceUpdateSecret(ns string, secretName string, secret *v1.Secret) error {
 	return nil
 }
 
+func ExecCmd(t *testing.T, client kubernetes.Interface, podName string, namespace string,
+	command string, stdin io.Reader) (string, string, error) {
+	cmd := []string{
+		"/bin/bash",
+		"-c",
+		command,
+	}
+	req := client.CoreV1().RESTClient().Post().Resource("pods").Name(podName).
+		Namespace(namespace).SubResource("exec")
+	option := &corev1.PodExecOptions{
+		Command: cmd,
+		Stdin:   true,
+		Stdout:  true,
+		Stderr:  true,
+		TTY:     false,
+	}
+	if stdin == nil {
+		option.Stdin = false
+	}
```

Comment:
Minor. Maybe it would be better to do it the other way around. Set `Stdin` to false by default and if there's some then set it to true.

Reply:
Thanks for your comment, I should do that on master version, but I don't want to apply this change here, as it's not in 4.6 yet

Reply:
OK then :)